webdataset/fluid.py

```diff
@@ -16,40 +16,54 @@
 from . import autodecode
 from . import iterators
 from . import tariterators
-from . import dataset
+from .dataset import (
+    WebDataset,
+    split_by_worker,
+    default_cache_dir,
+    default_cache_name,
+    default_cache_size,
+    default_cache_verbose
+)
 from .utils import reraise_exception
 
 
-
 class Dataset(IterableDataset):
     def __init__(
         self,
         urls,
         *,
         length=True,
-        splitter=dataset.split_by_worker,
+        splitter=split_by_worker,
         handler=reraise_exception,
         shuffle=False,
+        cache_dir=default_cache_dir,
+        cache_size=default_cache_size,
+        cache_name=default_cache_name,
+        cache_verbose=default_cache_verbose
     ):
         super().__init__()
-        self.dataset = dataset.WebDataset(
+        self._dataset = WebDataset(
```

Comment:
Was getting "Field duplicates the name of its containing class" in deepsource in previous commits, so I updated to use `self._dataset`. Not sure why it wasn't failing previously...anyways, I'm down to revert this if you want to ignore that error (`PTC-W0052`) instead.